app/policies/application_policy.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# Base policy class
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
```

Comment:
This guard should be the first line in the method:
```
# TODO: use I18n.t() for 'must be logged in' message
    raise Pundit::NotAuthorizedError, "must be logged in" unless user
```

Reply:
I didn't add it as we already make sure [only authenticated users can access the API](https://github.com/pieforproviders/pieforproviders/blob/develop/app/controllers/api/v1/api_controller.rb#L5). But it doesn't hurt adding the guard I guess.

Reply:
We don't have to bother translating the message since users will never get to see it.